README.md

```diff
@@ -706,24 +706,93 @@ _[see the values documentation for all configuration options](https://github.com
 ```console
 $ helm upgrade --install --namespace actions-runner-system --create-namespace \
              --wait actions-runner-controller actions-runner-controller/actions-runner-controller \
-             --set "githubWebhookServer.enabled=true,githubWebhookServer.ports[0].nodePort=33080"
+             --set "githubWebhookServer.enabled=true"
```

Comment:
```suggestion
             --set "githubWebhookServer.enabled=true,service.type=NodePort,githubWebhookServer.ports[0].nodePort=33080"
```

I prefer leaving NodePort as a first option introduced this document, so that K8s beginners can hopefully see you usually need to expose the nodeport anyway, either for the ingress controller or ARC, to let GitHub successfully send webhooks to your webhook endpoint.

Reply:
There is only one thing that is puzzling me about this, I was under the impression that GitHub will try to use SSL, and the "standard" way of setting up SSL on k8s is with an ingress. 

By exposing a node port the expectation is that the user will have to set up SSL on its own using something outside k8s?

That said I have no problem applying this change, but I am still curious.

Reply:
Change applied, but I am still curious =P

Reply:
> By exposing a node port the expectation is that the user will have to set up SSL on its own using something outside k8s?

@VinGarcia Mostly! I tend to provision an external load balancer outside of the K8s cluster, so that it can provide us a stable hostname and TLS termination. I'm usually an AWS shop so I often use AWS ALB for that. NodePort is used to provide access from the load balancer to nodes. In that case you don't even need to setup an ingress controller.

In many cases, it's almost about which component you'd like to expose via NodePort. If you use e.g. ingress-nginx as an ingress controller, you'd need to expose the ingress-controller pods via a NodePort service first, so that your load balancer can finally expose it to the internet(or set `type: LoadBalancer` so that it becomes a Service LoadBalancer, but under the hood it still use NodePort and the difference is that the k8s cloud provider creates an external loadbalancer for you...).

Also note that there are some advanced CNI plugins like https://github.com/aws/amazon-vpc-cni-k8s which are capable of connecting pods, nodes, and load balancers in a flat network. With that, you don't even need NodePort or Ingress. Your external loadbalancer can directly access your pods.

Reply:
I see, thanks for the detailed explanation @mumoshu =]. I am still learning a lot about k8s, so thanks for sharing.